koiki/sender.py

```diff
@@ -0,0 +1,15 @@
+class Sender():
+
+    """Encapsulates the sender information to be provided to all Koiki deliveries"""
+
+    def to_dict(self):
+        return {
+            'nombreRemi': 'La Zona',
+            'direccionRemi': 'C/ La Zona, 1',
+            'codPostalRemi': '08186',
+            'poblacionRemi': 'Barcelona',
+            'provinciaRemi': 'Barcelona',
+            'paisRemi': 'ES',
+            'emailRemi': 'lazona@opcions.org',
+            'telefonoRemi': '+34518888191',
```

Comment:
Can we move these content to a configuration file or envvars?

Reply:
I don't know if that's the address we'll use. I need to validate it with Sergi.